att_r: 0
att_p: 0
att_y: 0
thrust_sp: 0
